x-pack/auditbeat/module/system/package/schema.fbs

```diff
@@ -0,0 +1,20 @@
+namespace schema;
+
+ table Package  {
+	name:string;
+	version:string;
+	release:string;
+	arch:string;
+	license:string;
+	installtime:ulong;
```

Comment:
Can we get a comment or field naming change that indicates how this value is stored. Is this installtime stored in nanoseconds since epoch?

Reply:
It's stored in seconds since epoch not nano seconds.